### PR TITLE
Implement new ColumnedContentSection section

### PR DESCRIPTION
**Pull request by grosbouddha**
Implement new 3 column section used on home, home bus and pro:

TESTED=Locally in storybook and unit tests.

image
image
image
image

### CODE REVIEW

File: src/layout/section/columnedContentSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    line-height: ${font.m.lineHeight};
+    padding-top: ${space.m};
+  }
+
+  & .kirk-columned-content-section-media-cover img {
+    border-radius: 4px;
+    width: 100%;
+    height: 300px;
+    object-fit: cover;
+    border-radius: ${radius.s};
```
Comment (Tuxosaurus) [inline src/layout/section/columnedContentSection/index.tsx]: `border-radius` declared twice

File: src/layout/section/columnedContentSection/index.unit.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { mount } from 'enzyme'
+// import renderer from 'react-test-renderer'
```
Comment (gabrielagabriel) [inline src/layout/section/columnedContentSection/index.unit.tsx]: commented line

File: src/layout/section/columnedContentSection/index.tsx (hunk 1 above)
Comment (grosbouddha) [reply src/layout/section/columnedContentSection/index.tsx]: Done

File: src/layout/section/columnedContentSection/index.unit.tsx (hunk 2 above)
Comment (grosbouddha) [reply src/layout/section/columnedContentSection/index.unit.tsx]: Done

**Comment by grosbouddha** [comment]
PTAL

**Pull request merged**